inference-engine/ie_bridges/python/src/openvino/offline_transformations/offline_transformations_api_impl.cpp

```diff
@@ -56,6 +58,13 @@ void InferenceEnginePython::GenerateMappingFile(InferenceEnginePython::IENetwork
     manager.run_passes(network.actual->getFunction());
 }
 
+void InferenceEnginePython::CompressModelTransformation(InferenceEnginePython::IENetwork network) {
+    ngraph::pass::Manager manager;
```

Comment:
I think it is nice to have a comment describing a pipeline: nodes marking and inserting rt info about compression.